app/src/main/java/org/p2p/wallet/claim/ui/dialogs/ClaimInfoBottomSheet.kt

```diff
@@ -0,0 +1,88 @@
+package org.p2p.wallet.claim.ui.dialogs
+
+import androidx.fragment.app.FragmentManager
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import org.p2p.uikit.utils.setTextColorRes
+import org.p2p.wallet.R
+import org.p2p.wallet.claim.model.ClaimDetails
+import org.p2p.wallet.claim.model.ClaimFee
+import org.p2p.wallet.common.ui.bottomsheet.BaseDoneBottomSheet
+import org.p2p.wallet.databinding.DialogClaimInfoBinding
+import org.p2p.wallet.databinding.ItemClaimDetailsPartBinding
+import org.p2p.wallet.databinding.ItemInfoImageDoubleTextBinding
+import org.p2p.wallet.utils.args
+import org.p2p.wallet.utils.withArgs
+
+private const val ARG_CLAIM_DETAILS = "ARG_CLAIM_DETAILS"
+
+class ClaimInfoBottomSheet : BaseDoneBottomSheet() {
+
+    companion object {
+        fun show(
+            fm: FragmentManager,
+            claimDetails: ClaimDetails
+        ) {
+            ClaimInfoBottomSheet().withArgs(
+                ARG_CLAIM_DETAILS to claimDetails
+            ).show(
+                fm, ClaimInfoBottomSheet::javaClass.name
+            )
+        }
+    }
+
+    private lateinit var binding: DialogClaimInfoBinding
```

Comment:
override fun onDestroyView() {
    super.onDestroyView()
    _binding = null
}

Reply:
https://developer.android.com/topic/libraries/view-binding

Reply:
wow wow
GC does not work for that?
we did not do something like that ever